irohad/torii/impl/status_bus_impl.cpp

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "torii/impl/status_bus_impl.hpp"
+
+namespace iroha {
+  namespace torii {
+    StatusBusImpl::StatusBusImpl()
+        : worker_(rxcpp::observe_on_new_thread()), subject_(worker_) {}
```

Comment:
Why we create a thread each time on object creation? Better, in terms of testability and maintainability, to pass thread as an dependency.

Reply:
It won't help at all
For module tests it's possible to use mock instead
For integration test it's impossible to change this particular module